classes/abstracts/ActionScheduler_Abstract_ListTable.php

```diff
@@ -644,9 +727,9 @@ public function process_actions() {
 		$this->process_bulk_action();
 		$this->process_row_actions();
 
-		if ( ! empty( $_REQUEST['_wp_http_referer'] ) ) {
+		if ( ! empty( $_REQUEST['_wp_http_referer'] && ! empty( $_SERVER['REQUEST_URI'] ) ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Comment:
Order of parentheses needs to be tweaked:

```suggestion
		if ( ! empty( $_REQUEST['_wp_http_referer'] ) && ! empty( $_SERVER['REQUEST_URI'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```